extensions/amp-base-carousel/0.1/carousel.css

```diff
@@ -85,16 +85,17 @@
   overflow-x: hidden;
 }
 
-.i-amphtml-carousel-scroll[snap="false"] {
+.i-amphtml-carousel-scroll[snap='false'] {
   scroll-snap-type: none;
 }
 
-.i-amphtml-carousel-scroll[user-scrollable="false"] {
+.i-amphtml-carousel-scroll[user-scrollable='false'] {
   overflow: hidden;
 }
 
 .i-amphtml-carousel-spacer {
-  z-index: -1;  
+  visibility: hidden;
```

Comment:
Does this carry an accessibility concern? I can't imagine it does, but just want to verify.

Reply:
Confirming there is no impact on accessibility with this change.